charts/prism/values.yaml

```
@@ -682,3 +696,61 @@ deployments:
+argoRollouts:
+  enabled: true
+  controller:
+    replicaCount: 1
+    metrics:
+      enabled: true
+      serviceMonitor:
+        enabled: false
+  installCRDs: true
+  dashboard:
+    enabled: true
+  fullnameOverride: argo-rollouts
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)